src/core/client_channel/client_channel_filter.h

```diff
@@ -460,6 +459,7 @@ class ClientChannelFilter::LoadBalancedCall
   std::unique_ptr<LoadBalancingPolicy::SubchannelCallTrackerInterface>
       lb_subchannel_call_tracker_;
   grpc_call_context_element* const call_context_;
+  Arena* const arena_;
```

Comment:
Why change this to store a pointer in the base class?  Previously, the code was storing this only in the `FilterBasedLoadBalancedCall` subclass, not in the `PromiseBasedLoadBalancedCall` subclass, because the latter could get it from context.  Has something changed that makes that story no longer work?

Note that if we do stick with this change, then we should remove the `arena_` data member from `FilterBasedLoadBalancedCall` (line 552 below).

Reply:
This is simpler (and we don't care about PromiseBasedLoadBalancedCall anymore -- indeed I have the PR ready to remove that too)